projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -182,10 +185,10 @@ export class CartItemListComponent {
             value.entryNumber,
             value.quantity
           );
-        } else if (value && this.cart?.cartId && this.cart?.userId) {
+        } else if (value && this.cartId && userId) {
```

Comment:
The longer I look it seems like the takeUserId + subscription would be better solution here.

Reply:
fixed